agent/engine/interface.go

```diff
@@ -50,6 +52,8 @@ type TaskEngine interface {
 
 	Version() (string, error)
 
+	Info() (types.Info, error)
```

Comment:
i don't think this needs to be part of the TaskEngine interface, considering that this is used by the engine itself and doesn't need to be public

Reply:
Agreed, removed it from askEngine interface.